include/seastar/kafka/producer/kafka_producer.hh

```diff
@@ -0,0 +1,59 @@
+/*
+ * This file is open source software, licensed to you under the terms
+ * of the Apache License, Version 2.0 (the "License").  See the NOTICE file
+ * distributed with this work for additional information regarding copyright
+ * ownership.  You may not use this file except in compliance with the License.
+ *
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+/*
+ * Copyright (C) 2019 ScyllaDB Ltd.
+ */
+
+#pragma once
+
+#include <string>
+
+#include "../../../../src/kafka/connection/connection_manager.hh"
```

Comment:
Are those ../../../.. needed? They look strange.

Reply:
Also in all the other files.

Reply:
I don't really know how else we could refer to those files so they are visible to the linker

Reply:
Unless I have something set up in a wrong way

Reply:
The issue is that you're trying to include an internal header (one put directly in `src/kafka`) into an external, user-visible header (put in `include/seastar`). The solution is simply to move `connection/connection_manager.hh` in the `include/seastar/kafka/` directory, where it belongs. Otherwise, when a developer installs seastar and tries to `#include` this header, the preprocessor will also try to include a `"../../../../src/kafka/connection/connection_manager.hh"` path, which likely does not exist at all on the developer's PC.